### PR TITLE
Add JVC Projector Integration

### PR DESCRIPTION
**Pull request by iloveicedgreentea**
## Proposed change
This adds the documentation for my JVC Projectors integration. 

https://github.com/home-assistant/core/compare/dev...iloveicedgreentea:jvc_projector?expand=1

PR to the HA repo pending until this is made.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/67358
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by iloveicedgreentea** [comment]
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description. (message by DocsTargetBranch)

PR is here, added to description: https://github.com/home-assistant/core/pull/67358

File: source/_integrations/jvc_projectors.markdown [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+If using an NZ model, you MUST set a network password before using this.
+
+</div>
+
+{% configuration_basic %}
+host: description: IP address of the device, e.g., 192.168.1.32
+name: friendly name to use as the entity name. Will appear as remote.{name}
+password: If NZ model, the password you configured
+{% endconfiguration_basic %}
```
Comment (klaasnicolaas) [inline source/_integrations/jvc_projectors.markdown:40]: {% include integrations/config_flow.md %}

File: source/_integrations/jvc_projectors.markdown [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+#### Service `remote.get_command`
+
+Send a supported command to the projector. Refer to the library for supported commands. _A list of commands can be provided instead of a single command at a time. This has major performance improvements as it reuses the connection._
+{% raw %}
```
Comment (klaasnicolaas) [inline source/_integrations/jvc_projectors.markdown]:
```suggestion
```

File: source/_integrations/jvc_projectors.markdown [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+data:
+  command:
+    - "menu,menu"
+    - "menu,left"
+    - "menu,menu"
+target:
+  entity_id: remote.nz7
+```
+
+{% endraw %}
```
Comment (klaasnicolaas) [inline source/_integrations/jvc_projectors.markdown]:
```suggestion
```

File: source/_integrations/jvc_projectors.markdown [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+#### Service `jvc_projectors.sdr_picture_mode`
+
+Set optimal settings for Film when an SDR signal is present.
+
+| Service data attribute | Optional | Description                                                 |
+| ---------------------- | -------- | ----------------------------------------------------------- |
+| `entity_id`            | no       | Name of entity to send command to. For example `remote.nz7` |
+
+[jvc projectors]: /integrations/jvc_projectors
```
Comment (klaasnicolaas) [inline source/_integrations/jvc_projectors.markdown]:
```suggestion
```

File: source/_integrations/jvc_projectors.markdown [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+ha_release: 2022
+ha_domain: jvc_projectors
+ha_codeowners:
+  - "@iloveicedgreentea"
+ha_config_flow: false
+ha_platforms:
+  - remote
+---
+
+The `jvc_projectors` platform allows you to control [JVC Projectors](https://www.us.jvc.com/projectors/) from Home Assistant. It is likely that your device is supported by the [jvc_projectors] platform.
```
Comment (klaasnicolaas) [inline source/_integrations/jvc_projectors.markdown]:
```suggestion
The `jvc_projectors` integration allows you to control [JVC Projectors](https://www.us.jvc.com/projectors/) with Home Assistant.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**